docs/user-docs/latest/plugins/introduction-to-plugins.md

```diff
@@ -6,15 +6,33 @@ canonicalUrl: https://docs.strapi.io/user-docs/latest/plugins/introduction-to-pl
 
 # Introduction to plugins
 
-Strapi is built around different types of plugins. Core plugins are essential for your Strapi application to function, and therefore cannot be deactivated. Pre-installed plugins, such as Internationalization, are installed in a default Strapi application. Additional plugins that extend and customize Strapi applications are available in the [Marketplace](../plugins/installing-plugins-via-marketplace.md).
+Strapi is built around different types of plugins. There is a set of core plugins that are pre-installed in a default Strapi application and cannot be deactivated as they are essential for your Strapi application to function. These core plugins are:
+
+* Content Manager
+* Content Type Builder
+* Email
+* Media Library
+* Internationalization
+* Roles and Permissions
 
 ::: note
-Core plugins which include the Content Manager and the Content-type Builder are documented in their own sections of the user guide as they are quite specific (see [Introduction to the Content Manager](../content-manager/introduction-to-content-manager.md) and [Introduction to the Content-type Builder](../content-types-builder/introduction-to-content-types-builder.md)). This Plugins section focuses on how to manage plugins in general and provides documentation for the other, non-core plugins.
+Core plugins such as the Content Manager and the Content-type Builder are documented in their own sections of the user guide (see [Introduction to the Content Manager](../content-manager/introduction-to-content-manager.md) and [Introduction to the Content-type Builder](../content-types-builder/introduction-to-content-types-builder.md)). This section focuses on how to manage plugins in general and provides documentation for other, non-core plugins.
 :::
```

Comment:
I understand what you wanted to do here but I'm not sure we should focus that heavily on core plugins because the most important ones are not even documented in that section of the user guide, and also we've had several discussions with the Expansion squad about renaming types of plugins and getting rid of the "core plugin" name.
I think what's important to explain in that introduction is that Strapi is built around plugins and that this section lists and explains Strapi plugins, and how to manage them through the in-app Marketplace. We should still mention core plugins but not that heavily (that's why we initially opted for the note about core plugins).
So maybe we could rewrite an introduction that doesn't entirely focus on core plugins, and improve the note about core plugins (we could keep that bullet list you wrote and be more precise). What do you think?

Reply:
It's much better, thank you! 👏